src/main/java/flight/reservation/payment/Paypal.java

```diff
@@ -1,13 +1,52 @@
 package flight.reservation.payment;
 
+import flight.reservation.order.FlightOrder;
+
 import java.util.HashMap;
 import java.util.Map;
 
-public class Paypal {
+public class Paypal extends PaymentMethod {
     public static final Map<String, String> DATA_BASE = new HashMap<>();
 
     static {
         DATA_BASE.put("amanda1985", "amanda@ya.com");
         DATA_BASE.put("qwerty", "john@amazon.eu");
     }
+
+    final String email;
+    final String password;
+
+
+   public Paypal(String email, String password)  {
+       super();
+       this.email = email;
+       this.password = password;
+   }
+
+
+    public boolean processOrder(FlightOrder flightOrder) throws IllegalStateException {
```

Comment:
the `processOrder()` method could contain less duplicate code by factoring out the duplicate code